tsconfig.json

```diff
@@ -11,7 +11,6 @@
         "typeRoots": ["node_modules/@0x/typescript-typings/types", "node_modules/@types"],
         "strict": true,
         // These settings are required for TypeScript project references
-        "composite": true,
```

Comment:
Get bizarre tsc build errors with this enabled. Doesn't seem to affect anything to leave it out?